component-classes/index.js

```diff
@@ -93,7 +93,7 @@ export const pill = {
   label: 'pl-12 py-8 rounded-l-full',
   labelWithoutClose: 'pr-12 rounded-r-full',
   labelWithClose: 'pr-2',
-  close: 'pr-12 pl-4 py-10 rounded-r-full',
+  close: 'pr-12 pl-4 pt-4 pb-6 rounded-r-full !text-m',
```

Comment:
Did it work with `!text-m`? I thought the `!` char should be at the end like `text-m!` 👀

Reply:
It works with `!text-m`, but it seems to also work with `text-m!`. When I inspect the css on the browser it still translates it to `!important`. I just thought it was the same is in Tailwind, where they put the `!` before. But I can move it to the end? :)

Reply:
Oh, I wasn't aware that it's like that in Tailwind! [UnoCSS examples](https://github.com/unocss/unocss/blob/78d1d03b461288b472b8c71e0b23d8e4f07c8245/test/scope.test.ts#L10) also seem to follow that pattern. Good that it works regardless of where `!` is put 😄 Then I'd say keep it! Maybe we should in fact update all other examples to follow the Tailwind way one day ;)